### PR TITLE
redirect to home.

### PR DESCRIPTION
**Pull request by dhatch**

### CODE REVIEW

**Review by gj — APPROVED** [review]
verified locally; LGTM

File: docs/themes/oso-tailwind/layouts/partials/sidebar.html [hunk 1]
```diff
@@ -27,7 +27,7 @@
                 {{ $currentNode := . }}
                 {{ if eq $currentNode.Section "new-roles" }}
                 <ul>
-                    <li><a href="{{ .Site.Home.Permalink }}">Back to documentation home</a></li>
+                    <li><a href="/">Back to documentation home</a></li>
```
Comment (samscott89) [inline docs/themes/oso-tailwind/layouts/partials/sidebar.html:30]:
```suggestion
                    <li><a href="{{ .Site.Home.Permalink }index.html">Back to documentation home</a></li>
```

alternate

Comment (gj) [reply docs/themes/oso-tailwind/layouts/partials/sidebar.html:30]: what's the benefit?

Comment (gj) [reply docs/themes/oso-tailwind/layouts/partials/sidebar.html:30]: another: https://github.com/osohq/oso/blob/e1f274fdda65c988201ba8a76a3937a99a2da327/docs/themes/oso-tailwind/layouts/partials/header.html#L16

Comment (samscott89) [reply docs/themes/oso-tailwind/layouts/partials/sidebar.html:30]: I think my version would redirect back to the same language index.

**Pull request merged**